### DISCUSSION TITLE
Dungeon quit function

### DISCUSSION THREAD

**Discussion by Sirfirelord69** [DiscordDungeons/Bugs#1056] [Ideas]
Allow us to quit a dungeon, forfeiting all loot, xp and gold.
`#+dungeon forfeit` giving a confirm dialog with ✅ , or maybe a format like `#+gdisband`, so u have to type `#+dungeon forfeit YES` or maybe `#+dforfeit YES` something along those lines

**Comment by Sven65** [comment]
The point of dungeons is to be hard. If you want to exit it without any loot, you can do so by dying.
